src/material-icons.json

```diff
@@ -33,6 +33,9 @@
     "_file_js": {
       "iconPath": "./../../icons/javascript.svg"
     },
+    "_file_jsreact": {
```

Comment:
`_file_react` already exists with this icon

Reply:
So I can map ``_file_react`` with ``javascriptreact`` "languageIds" field and remove ``_file_jsreact``